arcade/gui/widgets/layout.py

```diff
@@ -590,11 +586,6 @@ def min_size(child: UIWidget) -> Tuple[float, float]:
 
             max_height_per_row[row_num][col_num] = (shmn_h, row_span)
 
-            for row in child_sorted_row_wise[
-                row_num : row_num + row_span  # noqa: E203
-            ]:
-                row[col_num : col_num + col_span] = [child] * col_span  # noqa: E203
-
```

Comment:
What was this doing, and did removing it cause any ill effects?

Reply:
I'm guessing it's dead code copy-pasted from `def add`.  The same thing appears in `def add`, where it's actually used.